src/main/java/com/spotinst/sdkjava/model/ISpotOceanK8sVirtualNodeGroupRepo.java

```diff
@@ -0,0 +1,6 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.model.bl.ocean.kubernetes.K8sVirtualNodeGroup;
+
+public interface ISpotOceanK8sVirtualNodeGroupRepo extends IRepository<K8sVirtualNodeGroup, Void, String>{
```

Comment:
This class should be in repo folder. Please check other Ocean ECS LaunchSpec CRUD for reference.

Reply:
This Repo class uses package-private interface com.spotinst.sdkjava.model.IRepository.  It is better the keep the same level as IRepository